content/stats/views.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "View"
+weight: 3
+aliases: [/core-concepts/metrics/views]
+---
+
+- [View](#view)
+- [Parts of a view](#parts-of-a-view)
+- [Aggregations](#aggregations)
+
+### View
+
+Views are the coupling of an [Aggregation](#aggregations) applied to a [Measure](/stats/measure) and optionally [Tags](/tag).
+Views are the connection to [Metric exporters](/exporters).
+
+### Aggregations
+The following aggregation methods are supported:
+
+* **Count**: The count of the number of measurement points.
+* **Distribution**: Histogram distribution of the measurement points.
+* **Sum**: A sum up of the measurement points.
+* **LastValue**: Keeps the last recorded value, drops everything else.
+
+### Parts of a view
+
+Field|Description|Example|Restrictions
+---|---|---|---
+Name|Identifies the metric being collected|grpc.io/client/completed_rpcs|Make it a fully qualified and distinguishable name
```

Comment:
In Java and Go we do sanitizations on Prometheus metric names (i.e replace other characters with '_'), though I agree it's better to use the supported names in the example.

Reply:
Thanks for the inputs @mayurkale22 @songy23. I disagree about changing them, that is because that should be an implementation for only the exporter. OpenCensus Python Prometheus just doesn't seem to have sanitization and that shouldn't affect the way users communicate and that example is from live gRPC stats as expected by the specs https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/gRPC.md#default-views

Reply:
SGTM, can we fill issue to sanitize metric names in OC python Prometheus exporter.